pkg/k8shandler/visualization.go

```diff
@@ -122,6 +129,51 @@ func createOrUpdateKibanaDeployment(logging *logging.ClusterLogging) (err error)
 	return nil
 }
 
+func createOrUpdateOauthClient(logging *logging.ClusterLogging, oauthSecret string) (err error) {
+
+	clusterDomain, err := utils.GetClusterDNSDomain()
+	if err != nil {
+		return nil
+	}
+
+	oauthClient := &oauth.OAuthClient{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "OAuthClient",
+			APIVersion: oauth.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kibana-proxy",
+			Namespace: logging.Namespace,
+			Labels: map[string]string{
+				"logging-infra": "support",
+			},
+		},
+		Secret: oauthSecret,
+		// TODO: don't just use string concatination here...
+		RedirectURIs: []string{
+			"https://kibana." + clusterDomain,
```

Comment:
Isn't this an issue if user's customize their route?  (i.e. operations use of `logs.somedomain.com`).  Do we need to add a field to the CRD?

Reply:
I started having a discussion about this on slack with @richm. I think we would need pm to resolve this, I'm of the opinion that we should just control that, I don't see a benefit to letting them choose what their Kibana route is and it increases our complexity.

Reply:
I think for now, let's merge this PR, and revisit the issue in another PR